Dockerfile

```diff
@@ -51,7 +51,7 @@ ENV \
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
     export PATH=$PATH:$HOME/.cargo/bin && \
     scripts/init.sh && \
-    cargo build --locked --$PROFILE
+    cargo build --locked --$PROFILE --features on-chain-release-build
```

Comment:
If this flag is required now, I think we need to add it to `$PROFILE` variable. @krolol should know where it is.